model/src/main/kotlin/licenses/LicenseInfoResolver.kt

```diff
@@ -166,7 +166,7 @@ class LicenseInfoResolver(
         return processedCopyrightStatements.mapValues { (_, originalStatements) ->
             resolvedCopyrightFindings.filter { it.statement in originalStatements }
         }.filterValues { it.isNotEmpty() }.entries.map { (statement, findings) ->
-            ResolvedCopyright(statement, findings.toSet(), isGarbage = statement in copyrightGarbage.items)
```

Comment:
Passing garbage entries to the copyright statements processor implies that the garbage entries may have effect on the processed entries corresponding to raw non-garbage entries. This should be avoided IMO by not passing garbage to the processor.

Reply:
I have updated the PR as discussed offline.